github.go

```diff
@@ -222,3 +234,69 @@ func (c Client) ListReleases(org string, repos []string, daysCount int) ([]Relea
 	}
 	return listReleases, nil
 }
+
+func (c Client) IssueWithLabels(org string, repos []string, issueLabels []string, daysCount int) ([]IssueList, error) {
+	var issueList []IssueList
+
+	dayDiff := daysCount * -1
+
+	//get open issues to be worked on and which has not been assigned to someone
+	issueListOptions := &github.IssueListByRepoOptions{
+		State:    "open",
+		Assignee: "",
+		Labels:   issueLabels,
+		ListOptions: github.ListOptions{
+			PerPage: 20,
+		},
+	}
+
+	// Iterate over the repos
+	for _, repo := range repos {
+
+		var listIssues []github.Issue
+		issueDateReached := false
+
+		for {
+
+			issues, response, err := c.Client.Issues.ListByRepo(c.Context, org, repo, issueListOptions)
+			if err != nil {
+				return nil, err
+			}
+			log.Printf("Response: %v", response)
+			if response.StatusCode != http.StatusOK {
+				return nil, errors.New("Could not get the response for fetching issues")
+			}
+			for _, issue := range issues {
+
+				publishedDate := issue.GetCreatedAt()
+				startDate := time.Now().AddDate(0, 0, dayDiff)
+				log.Println("publishedDate", publishedDate, "start date", startDate, " if condition", publishedDate.Before(startDate))
+
+				if publishedDate.Before(startDate) {
```

Comment:
Q: The assumption with PR list was that it was listed in descending order of creation time. Does that hold true with this API as well?

Reply:
Added this search criteria explicitly